Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM cimg/node:14.17
+FROM cimg/node:18.13
```

Comment:
I was thinking if we should use `cimg/node:lts` instead?

Reply:
Yeah, if that works it should be better!

Reply:
It's better to not use `its` because this image is only built every now and then whenever this repository is updated.

This means that we will have a docker image in docker hub with `lts` in the name but the actual `node:lts` will be different as soon as NodeJS updates their docker image `its` tag.